src/Sulu/Bundle/MediaBundle/Tests/Features/collection_management.feature

```diff
@@ -16,32 +19,47 @@ Feature: Collection management
 
     Scenario: Edit collection
         Given the media collection "Foobar" exists
-        And I am on "/admin/#media/collections"
-        And I expect a thumbnail to appear
-        And I click on the element ".thumbnail"
+        And I click the "collections-edit" navigation item
+        And wait a second
+        And I click on the element ".data-navigation-item-thumb"
         And I expect the "husky.loader.initialized" event
         Then I should see "Foobar"
 
     Scenario: Delete collection
         Given the media collection "Foobar" exists
         And I am on the settings page for the media collection
-        And I click the trash icon
+        And I click toolbar item "delete"
         Then I expect a confirmation dialog to appear
         And I confirm
         # TODO: No notification: And I expect a success notification to appear
         # See: https://github.com/sulu-cmf/sulu/issues/708
         And I wait a second
         And I wait a second
-        Then the media collection "Foobar" should not exist
+        And the media collection "Foobar" should not exist
 
     Scenario: View collection
         Given the file "image1.png" has been uploaded to the "Dornbirn" collection
-        Given the file "image2.png" has been uploaded to the "Dornbirn" collection
-        Given the file "image3.jpg" has been uploaded to the "Dornbirn" collection
-        Given the file "image4.jpg" has been uploaded to the "Dornbirn" collection
+        And the file "image2.png" has been uploaded to the "Dornbirn" collection
+        And the file "image3.jpg" has been uploaded to the "Dornbirn" collection
+        And the file "image4.jpg" has been uploaded to the "Dornbirn" collection
         And I am on the media collection edit page
         Then I expect to see "4" ".item" elements
 
+    Scenario: Move collection
+        Given the media collection "Dornbirn" exists
+        And the media collection "Foobar" exists
+        And I am editing the media collection "Dornbirn"
+        And I click toolbar item "collection-move"
```

Comment:
`When I click` -- I only found out about `When` last week, but its actually pretty important for understanding scenarios. See https://github.com/phpcr/phpcr-shell/pull/141#discussion_r25599570